misc/git.talon

```diff
@@ -59,15 +58,26 @@ git reset soft: "git reset --soft "
 git reset hard: "git reset --hard "
 git restore: "git restore "
 git restore staged: "git restore --staged "
-git remote show origin: "git remote show origin\n"
+git restore source: "git restore --source="
```

Comment:
I don't think this `=` is intentional

Reply:
Actually maybe it is, sometimes one gets used and sometimes one doesn't
![image](https://user-images.githubusercontent.com/9077250/128291299-f55abf68-10ca-4aa4-8a78-5526fa806ad7.png)
![image](https://user-images.githubusercontent.com/9077250/128291302-d03ebd83-3d39-4716-8423-d3d3006845c4.png)